src/app/public/start-chapter/page.tsx

```diff
@@ -0,0 +1,56 @@
+"use client";
+
+import NewChapterForm from "@components/NewChapterForm";
+
+
+
+const startChapter = () => {
+  return (
+     <>
+  <div className = "bg-med-tan">
+      <div className="px-20">
+      <div className="py-3">
+        <div className="py-2 text-4xl font-bold">Start a Chapter</div>
+        <div className="pt-6 text-lg font-semibold">
+          Start a chapter of the Legacy Project, Inc. on your campus today!
+        </div>
+      </div>
+      <div>
+        <p className="py-3">
+          This form serves to identify student leaders who are interested in
+          starting a chapter of The Legacy Project, Inc. (TLP) at their US-based
+          college or university.{" "}
+        </p>
+        <p className="py-3">
+          TLP is a student-led organization that seeks to{" "}
+          <strong>
+            {" "}
+            foster intergenerational connections between students and older
+            adults
+          </strong>{" "}
+          in their local communities. Furthermore, TLP is dedicated to
+          documenting and amplifying the life stories of the older generation to
+          ensure that their legacies are preserved for years to come.
+        </p>
+        <p className="py-3">
+          We encourage all to apply, <strong> no experience necessary, </strong>{" "}
+          and we will be in contact soon to explore this opportunity.{" "}
+        </p>
+        <p className="py-3">
+          {" "}
+          For additional information, feel free to reach out at{" "}
+          <strong> exec@thelegacyproject.org </strong>
+        </p>
+      </div>
+    </div>
+    <div>
+    <NewChapterForm />
```

Comment:
I think we should move `<NewChapterForm /> to line 43 to add the padding. It looks a bit weird on my end

![image](https://github.com/JumboCode/TheLegacyProject2/assets/74647679/67fb11b2-cad5-43db-8057-91542d763c04)